Packages/vcs/Lib/VCS_validation_functions.py

```diff
@@ -2019,3 +2036,31 @@ def _setvp(self, value):
         maxvalue=1.)
     self._viewport = value
 viewport = property(_getvp, _setvp)
+
+
+def _getfillareaopacity(self):
+    return self._fillareaopacity
+
+
+def _setfillareaopacity(self, value):
+    if value is not None:
+        value = checkOpacitiesList(
+            self,
+            'fillareaopacity',
+            value)
+        self._fillareaopacity = value
+fillareaopacity = property(_getfillareaopacity, _setfillareaopacity)
+
+
+def _getfillareacolors(self):
+    return self._fillareacolors
+
+
+def _setfillareacolors(self, value):
+    if value is not None:
+        value = checkColorList(
+            self,
+            'fillareacolors',
+            value)
+    self._fillareacolors = value
```

Comment:
Okay. But that shouldn't be the case anymore, right? Since we have an opacity field now.